src/main/java/lotto/domain/WinningNumber.java

```diff
@@ -7,16 +7,22 @@
 public class WinningNumber {
     private final static String DEFAULT_REGEX = ",";
     private final List<Integer> winningNumbers;
+    private final int bonusNumber;

```

Comment:
```suggestion
    private final static String DEFAULT_REGEX = ",";
    
    private final List<Integer> winningNumbers;
    private final int bonusNumber;

```
상수와 인스턴스 변수는 빈라인으로 구분해주면 가독성이 좀더 좋아질 수 있어요! 😄

Reply:
인텔리제이에서 usages기능 때문에 자꾸 띄워진걸로 보이네요. 잘 확인하겠습니다!